src/templates/default/proposals/_proposal_create.html

```diff
@@ -15,3 +15,7 @@ <h3 class="dashboard-header">
   </div>
 </form>
 {% endblock main-content %}
+
+{% block extra_js %}
+<script src="{% static 'js/proposals/proposal_form.js' %}"></script>
+{% endblock %}
```

Comment:
nit: add a newline at the end of the file

Reply:
Resolved in this [commit](https://github.com/pycontw/pycon.tw/pull/983/commits/15e661a6295982954459d1c52e3005d611b79731)